File: POOPHP/app/Model/AreaUsuario.php
<?php
namespace App\Model;
class AreaUsuario
{
   public $idAreaUsuario="";
   public $idArea="";
   public $idUser="";
   public function __construct(String $idA,String $idU)
    {
        $this->establecerDatos($idA,$idU);
    }
    public function ObtenerIdArea()
    {
        return $this->idArea;
    }
    public function ObteneridUser()
    {
        return $this->idUser;
    }
    public function establecerDatos(String $idAr,String $idUs)
    {
        $this->idArea=$idAr;
        $this->idUser=$idUs;
    }
}
?>

File: POOJAVA/src/clasesPOO/AreaUsuario.java
package clasesPOO;

public class AreaUsuario {
    String idAreaUsuario;
    String idArea;
    String idUser;
    public AreaUsuario(String idA,String idU)
    {
        establecerDatos(idA,idU);
    }
    public String ObtenerIdArea()
    {
       return idArea;
    }
    public String ObteneridUser()
    {
       return idUser;
    }
    void  establecerDatos(String idAr,String idUs)
    {
        idArea=idAr;
        idUser=idUs;
    }
}
